### DISCUSSION TITLE
Laravel 5.7.5 API Request Json

### DISCUSSION THREAD

**Discussion by blpraveen** [laravel/framework#34440] [General]
I am using laravel API Authentication it is authenticating with api_token successfully as form-data. When I pass a data as Content_Type application/json. I am not receiving the Json data it shows empty array. How do I receive a JSON Request in the API?
![image](https://user-images.githubusercontent.com/3274289/93790744-02ef2900-fc51-11ea-99ae-21bd3fadb0bf.png)

	public function create_invoice(Request $request) {   
		$data = $request->all();
	}

**Comment by Vikrant659** [comment]
You have to do something like that:-

```
public function create_invoice(Request $request) {   
   $data = $request->all(); 
   $code=200;
   return response()->json(['status'=>true','statuscode'=>$code,'data'=>$data]);
}
```
